### ISSUE TITLE
Do you have pretrained weights which can be used in Python for shoe detection 

### ISSUE THREAD

**Issue by ayush12gupta** [ibaiGorordo/shoeDetection#1]
If available can you please provide them

**Comment by ibaiGorordo** [comment]
Hello,

I have added the weights to the repository. Note: The model is designed to be fast in mobile devices so the accuracy might not be as good as a bigger model.

I have also added an Google Colab inference example using Tensorflow 1. You can use it as a basis for your Python implementation.
[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ibaiGorordo/shoeDetection/blob/master/Python%20inference/Python_inference.ipynb)

Ibai
